### PR TITLE
更新页面排版微调

### PR DESCRIPTION
**Pull request by insualk**
移除了最高层级列表元素的字体加粗。

### CODE REVIEW

**Comment by ztl8702** [comment]
is this still up to date?

**Comment by insualk** [comment]
> is this still up to date?

Not now but there shouldn't be any problem merging.

File: web/src/app/help/help.component.html [hunk 1]
```diff
@@ -46,14 +46,14 @@ <h2>筛选流程</h2>
       </ol>
     </li>
     <li>
-      经过上述几轮筛选后的字词，按造字法与用字法，分为四种：本字、俗字、同音字、训读字。优先级为：本字<span class="space">&nbsp;</span>≈<span class="space">&nbsp;</span>俗字<span class="space">&nbsp;</span>><span class="space">&nbsp;</span>同音字<span class="space">&nbsp;</span>><span class="space">&nbsp;</span>训读字
+      经过上述几轮筛选后的字词，按造字法与用字法，分为四种：本字、俗字、同音字、训读字。优先级为：本字≈俗字>同音字>训读字
       <ol>
         <li>若使用优先级较高的用字会产生歧义，则不选择该写法。</li>
         <li>在本字与俗字中，优先选择学界有共识的、没有争议的本字与被民间普遍接受的俗字。若二者冲突，则以民间接受度为基准。</li>
-        <li>若只有同级比较，那么以<span class="space">&nbsp;</span>Unicode<span class="space">&nbsp;</span>码拓展集的先后顺序为优先。
+        <li>若只有同级比较，那么以<span class="space">&ensp;</span>Unicode<span class="space">&ensp;</span>码拓展集的先后顺序为优先。
```
Comment (ztl8702) [inline web/src/app/help/help.component.html:53]: Today I Learned: `ensp`

File: web/src/app/help/help.component.ts [hunk 2]
```diff
@@ -1,12 +1,14 @@
 import {Component, OnInit} from '@angular/core';
+// import {MatButtonModule} from '@angular/material/button';
```
Comment (ztl8702) [inline web/src/app/help/help.component.ts:2]: remove?

File: web/src/app/help/help.component.ts [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
 
 @Component({
   selector: 'app-help',
   templateUrl: './help.component.html',
-  styleUrls: ['./help.component.scss']
+  styleUrls: ['./help.component.scss'],
+  preserveWhitespaces: true
```
Comment (ztl8702) [inline web/src/app/help/help.component.ts:8]: why is this needed?

File: web/src/app/help/help.component.ts [hunk 4]
```diff
@@ -1,12 +1,14 @@
...
 })
 export class HelpComponent implements OnInit {
   constructor() {}
 
   ngOnInit() {}
-}
+}
```
Comment (ztl8702) [inline web/src/app/help/help.component.ts:14]: nit: prefer a newline at the end of the file

File: web/src/app/help/help.component.scss [hunk 5]
```diff
@@ -30,4 +34,18 @@ span.space, span.space[hidden] {
...
     font: 0.89em Arial;
-}
\ No newline at end of file
+}
+
+.mat-button {
+    padding-left: 0;
+    padding-right: 0;
+    line-height: 1em;
+    // color: /;/
```
Comment (ztl8702) [inline web/src/app/help/help.component.scss:43]: remove?

File: web/src/app/help/help.component.scss [hunk 6]
```diff
@@ -1,3 +1,7 @@
+.content {
+    font-family: 'Noto Serif SC', 'Noto Serif TC', 'Ext-0', 'Ext-1', serif;
```
Comment (ztl8702) [inline web/src/app/help/help.component.scss:2]: can we make this a global font style? (i.e. create a new class in web/src/styles.scss and reuse the class in the help component)

File: web/src/app/help/help.component.html (hunk 1 above)
Comment (insualk) [reply web/src/app/help/help.component.html:53]: There are actually some more: http://www.oicqzone.com/pc/2015083122336.html

File: web/src/app/help/help.component.ts (hunk 3 above)
Comment (insualk) [reply web/src/app/help/help.component.ts:8]: No, it's not. I was trying to find a way to keep the whitespace but this is actually not the problem.

File: web/src/app/help/help.component.ts (hunk 4 above)
Comment (insualk) [reply web/src/app/help/help.component.ts:14]: Any reason for that?

File: web/src/app/help/help.component.scss (hunk 5 above)
Comment (insualk) [reply web/src/app/help/help.component.scss:43]: Oh yes.

File: web/src/app/help/help.component.scss (hunk 6 above)
Comment (insualk) [reply web/src/app/help/help.component.scss:2]: Yea I was thinking about it but I kind of get confused by the the font settings in all the places... I guess we can talk about this this Sunday?

File: web/src/app/help/help.component.ts (hunk 2 above)
Comment (insualk) [reply web/src/app/help/help.component.ts:2]: yes.

File: web/src/app/help/help.component.ts (hunk 4 above)
Comment (ztl8702) [reply web/src/app/help/help.component.ts:14]: https://stackoverflow.com/a/729795 

Mostly historical reasons. For us this is just a stylistic issue. No big deal. I sometimes miss this as well.

File: web/src/app/help/help.component.scss (hunk 6 above)
Comment (ztl8702) [reply web/src/app/help/help.component.scss:2]: In general, we should limit the number of font styles in the app. So it makes sense to move all font styles into a centralised place.

Can you create an issue for this?

**Comment by insualk** [comment]
Any comments on the links?  @ztl8702

**Comment by ztl8702** [comment]
Looks good to me for now. 

We need to redesign typography (including link styles) in #532

**Comment by ztl8702** [comment]
Feel free to merge.

**Comment by ztl8702** [comment]
merge?

**Comment by ztl8702** [comment]
Thanks! (For future reference, when there is more than one approval on the PR, you can just remove the do-not-merge tag; and mergify bot will merge the PR when the tests / CI builds pass. No need to wait and merge manually.)

**Comment by ztl8702** [comment]
Thanks! (For future reference, when there is more than one approval on the PR, you can just remove the do-not-merge tag; and mergify bot will merge the PR when the tests / CI builds pass. No need to wait and merge manually.)
